dev/vagrant/bootstrap/vault/01-create_vault_policy_to_read_secrets.yml

```
@@ -0,0 +1,12 @@
+- name: Create policy in vault to read secrets
+  shell: |
+    vault policy write kv-secret - << EOF
+    path "secret/minio" {
```

Can you create a nested path for `minio` object?
```suggestion
    path "secret/data/minio" {
```

---

`vault_kv_path` also needs to be corrected, but **not** in the playbook.